apis/r/tests/testthat/test_SOMAReader-Iterated.R

```diff
@@ -15,7 +15,7 @@ test_that("Iterated Interface from SOMAReader", {
     expect_true(dir.exists(uri))
 
     ctx <- tiledb_ctx()
-    sr <- sr_setup(uri, config=as.character(config(ctx)))
+    sr <- sr_setup(uri, config=as.character(tiledb::config(ctx)))
```

Comment:
Or call `library(tiledb)` ?  It is a dependency of the package...

Reply:
Witness:

```sh
edd@rob:~/git/tiledb-soma/apis/r/tests/testthat(main)$ grep library *
test_SOMAReader-Arrow.R:    library(arrow)
test_SOMAReader-Arrow.R:    library(tiledb)
test_SOMAReader-Iterated.R:    library(arrow)
test_SOMAReader-Iterated.R:    library(bit64)
test_SOMAReader-Iterated.R:    library(data.table)
test_SOMAReader-Iterated.R:    library(dplyr)
test_SOMAReader-Iterated.R:    library(tiledb)
edd@rob:~/git/tiledb-soma/apis/r/tests/testthat(main)$ 
```

Reply:
So given this line you really do not need this:

https://github.com/single-cell-data/TileDB-SOMA/blob/f9e29883c7de5e4c93b3b0a68a3fe69fcceee362/apis/r/tests/testthat/test_SOMAReader-Iterated.R#L8

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/1084